### ISSUE TITLE
Allow a fader for Master track on Mixer

### ISSUE THREAD

**Issue by milkmiruku** [j3ffhubb/musikernel#10]
To allow controlling the level for the whole track.

**Comment by j3ffhubb** [comment]
That is a very reasonable request (in fact, I'm now wondering why I never thought of that), I'll put it on my TODO list.

**Comment by milkmiruku** [comment]
I realise now there is the volume pot in the transport area, though I guess this is for monitoring level and wouldn't affect the render level.

**Comment by j3ffhubb** [comment]
Yes, that is correct.  What you can do as a workaround is add a fader plugin (Simple Fader or MK Channel) to the master track's plugins, but obviously it would be desirable to see it on the mixer instead of having to switch to the "Plugin Rack" tab to change the volume for the master track.
